android/java/res/values/brave_colors.xml

```diff
@@ -393,4 +393,10 @@
     <color name="text_secondary">#3f4e55</color>
     <color name="primary_20">#d5dcff</color>
     <color name="button_background">#3F39E8</color>
+    <color name="primitive_primary_20">#d5dcff</color>
+    <color name="primitive_primary_30">#b7c6ff</color>
+    <color name="primitive_primary_70">#3b08c2</color>
+    <color name="primitive_primary_80">#2a118e</color>
```

Comment:
do we have same colors in both light and dark modes?

Reply:
we user the same colors for light/dark mode. we don't have specific dark colors for primitive colors.